azure-ai-search/tests/test_azure_search.py

```diff
@@ -38,13 +37,35 @@ def test_transform_result(self):
             "url": "http://example.com",
             "score": 1.23,
             "title": "Example Title",
-            "content": "Example content",
+            "content": "Highlighted content",
             "subtitle": "Example Subtitle",
             "last_updated": "2023-01-01T00:00:00Z",
         }
         transformed_result = transform_result(mock_result)
         self.assertEqual(transformed_result, expected_output)
 
+    def test_transform_result_no_highlights(self):
+        mock_result_no_highlights = {
+            "id": "123",
+            "url": "http://example.com",
+            "@search.score": 1.23,
+            "metadata_storage_name": "Example Title",
+            "@search.highlights": None,
+            "subtitle": "Example Subtitle",
+            "metadata_last_modified": "2023-01-01T00:00:00Z",
+        }
+        expected_output_no_highlights = {
+            "id": "123",
+            "url": "http://example.com",
+            "score": 1.23,
```

Comment:
are search score coming from AI Search not normalized between 0 and 1?

Reply:
I believe they are unbounded. Check https://learn.microsoft.com/en-us/azure/search/index-similarity-and-scoring#scoring-algorithms-used-in-full-text-search

Reply:
ok, but then within the particular set of results you return you could compute doc_score/(max(doc_score) - min(doc_score))?

we have to think also about integrating the other signals in the postgresql database to rerank the results.

Reply:
(probably a separate issue to work on this)

Reply:
Here: https://github.com/ai-cfia/azure-db/issues/8